src/cookieSync.js

```diff
@@ -93,7 +119,51 @@ function ajax(url, callback, data, options = {}) {
   }
 }
 
-// Send empty data to receive cookie sync status for all prebid server adapters.
-// In next phase we will read placement id's from query param and will only get cookie sync status of bidders participating in auction
-var data = '{}';
-ajax(ENDPOINT, process, data);
\ No newline at end of file
+function getUrlParam(paramName) {
```

Comment:
to read two params from URL using this function, one will need to call this function twice and the same URL processing will be done twice, so change this function to return all URL params in an object, then you can call this function once and check required params from the output.

Reply:
Move this function to util

Reply:
Done